libraries/helpers.rb

```diff
@@ -32,7 +32,7 @@ def vagrant_platform_package(vers = nil)
     case node['platform_family']
     when 'debian'
       "vagrant_#{vers}_x86_64.deb"
-    when 'redhat', 'fedora'
```

Comment:
this change is not just removing `'redhat'`

See here for example: https://github.com/jtimberman/vagrant-cookbook/pull/19/files#diff-38a4aeddb4e620724d3b85362cb8b939R26